src/main/webapp/mapScript.js

```diff
@@ -1,5 +1,5 @@
 // Global Variables
-const API_KEY = 'AIzaSyBf5E9ymEYBv6mAi78mFBOn8oUVvO8sph4';
+const API_KEY = displayKey();
```

Comment:
You'll want to generate a fresh API key if you haven't already; this one's compromised (hosted in a public repo).